spark/src/test/scala/org/apache/spark/sql/delta/InCommitTimestampSuite.scala

```diff
@@ -58,6 +61,17 @@ class InCommitTimestampSuite
     commitInfo.get.inCommitTimestamp.get
   }
 
+  private def getFileModificationTimesMap(
+      deltaLog: DeltaLog, start: Long, end: Long): Map[Long, Long] = {
+    deltaLog.store.listFrom(
+        FileNames.listingPrefix(deltaLog.logPath, start), deltaLog.newDeltaHadoopConf())
+      .collect {
+        case FileNames.DeltaFile(fs, v) =>
+          v -> fs.getModificationTime
+      }.takeWhile(_._1 <= end)
+      .toMap
```

Comment:
```suggestion
      .collect { case FileNames.DeltaFile(fs, v) => v -> fs.getModificationTime }
      .takeWhile(_._1 <= end)
```

Reply:
I have updated the PR with all of the suggested changes